source/chapters/appendix/mixxx_controls.rst

```diff
@@ -2506,6 +2506,19 @@ Any control listed above for :mixxx:cogroupref:`[ChannelN]` will work for a samp
 
    .. versionadded:: 2.0.0
 
+.. mixxx:control:: [ChannelN],update_replaygain_from_pregain
+                   [PreviewDeckN],update_replaygain_from_pregain
+                   [SamplerN],update_replaygain_from_pregain
+
+  Applies the deck pregain knob value to the detected ReplayGain value for the
+  current track. This is a way to update the ReplayGain value of a track if it
+  has been detected incorrectly. When this control is triggered, the pregain
+  value for the deck will be centered so that there is no audible difference in
+  track volume, so this operation is safe to use during performance.
```

Comment:
```suggestion
  track volume, so this operation is safe to use during performance, if the controller mapping uses soft-takeover for the pregain knob.
```

Reply:
Hmm, this always applies. Otherwise even adjusting the gain knob via mouse will lead to issues, regardless of replay gain. So I IMHO this addition is not really needed.

Reply:
This is a long sentence already. Also, a controller is not necessarily the default.
What about
"If you use a controller make sure to enable SoftTakeover for the pregain knob for full comfort"

Reply:
> Hmm, this always applies. Otherwise even adjusting the gain knob via mouse will lead to issues, regardless of replay gain. So I IMHO this addition is not really needed.

true. let's leave it as is.

Reply:
This is the documentation page for developers of controller mappings. And this is the point, which needs to be considered in a mapping, because all other COs can be used safely without implemnting soft-takeover, but this one not.

Reply:
I think it's fine to add this clarification